.editorconfig

```
@@ -5,6 +5,24 @@ end_of_line = crlf
 charset = utf-8
 indent_style = space
 indent_size = 4
+dotnet_style_operator_placement_when_wrapping = beginning_of_line
+tab_width = 2
```

Just so general text editor settings are together and not mixed in with C# ones.

```suggestion
tab_width = 2
dotnet_style_operator_placement_when_wrapping = beginning_of_line
```